Core.DomainServices/Queries/Organization/QueryByNameOrCvrContent.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Linq;
+
+namespace Core.DomainServices.Queries.Organization
+{
+    public class QueryByNameOrCvrContent : IDomainQuery<DomainModel.Organization.Organization>
```

Comment:
Remember to add a unit test of this class alone (check the other unit tests for domain queries)

right now you are "testing it in action" but not in isolation.